tests/lib/rules/require-optimization.js

```diff
@@ -131,10 +131,19 @@ ruleTester.run('react-require-optimization', rule, {
         }
       }
     `,
-    parser: parsers.BABEL_ESLINT,
-    errors: [{
-      message: MESSAGE
-    }]
+    parser: parsers.BABEL_ESLINT
+  }, {
+    code: `
+    import React from "react";
+    class YourComponent extends React.Component {
+      handleClick = () => {}
+      shouldComponentUpdate = () => true;
```

Comment:
why is this desirable? lifecycle methods should never be implemented in this way, only as proper intance methods.

Reply:
Ok, I've figured out, that it's not desirable.